pom.xml

```diff
@@ -85,7 +85,7 @@
     <dependency>
       <groupId>com.facebook.presto</groupId>
       <artifactId>presto-parser</artifactId>
-      <version>0.126</version>
+      <version>0.128</version>
```

Comment:
Turns out presto-parser is actually far far far far away from being a stable library!